src/library/scala/concurrent/Future.scala

```diff
@@ -9,12 +9,11 @@
 package scala.concurrent
 
 import scala.language.higherKinds
-
 import java.util.concurrent.{CountDownLatch, TimeUnit}
-import java.util.concurrent.atomic.AtomicInteger
+import java.util.concurrent.atomic.{AtomicInteger, AtomicReference}
 
 import scala.util.control.NonFatal
-import scala.util.{Try, Success, Failure}
+import scala.util.{Failure, Success, Try}
```

Comment:
+1 intellij or scalafmt needs an option for that, if they don't have it. But it would need heuristics to decide Success before Failure, maybe inspecting the implementations of `toEither` or `toOption` etc.

Reply:
must say I didn't modify this line, Intellij does its magic for me :sunglasses: